.github/workflows/deploy-gh-pages.yml

```diff
@@ -0,0 +1,71 @@
+name: Deploy to GitHub pages
+
+on:
+  push:
+    branches: [ "master" ]
+  workflow_dispatch:
+
+# Allow only one concurrent deployment, skipping runs queued between the run in-progress and latest queued.
+# However, do NOT cancel in-progress runs as we want to allow these production deployments to complete.
+concurrency:
+  group: "gh-pages"
+  cancel-in-progress: false
```

Comment:
This isn't strictly necessary, but it was shown in the documentation and seems like a good safeguard to ensure we don't get into a broken state if one action is interrupted and then the subsequent one fails, or something like that.